shapely/algorithms/cga.py

```diff
@@ -1,6 +1,29 @@
+from warnings import warn
+
 import numpy as np
 
 import shapely
+from shapely import _is_ccw_geos_3_7
+from shapely.decorators import vectorize_geom
+from shapely.errors import ShapelyDeprecationWarning
+from shapely.geos import geos_version
+
+__all__ = [
+    "is_ccw",
+    "force_ccw",
+    "is_ccw_impl",
```

Comment:
We don't want to expose this one top-level, I think

Reply:
OK, I'll remove this